fil_fungible_token/src/receiver/types.rs

```diff
@@ -15,9 +16,14 @@ pub trait FrcXXXTokenReceiver {
     fn token_received(params: TokenReceivedParams);
 }
 
-#[derive(Serialize_tuple, Deserialize_tuple)]
+#[derive(Serialize_tuple, Deserialize_tuple, PartialEq, Eq, Clone, Debug)]
 pub struct TokenReceivedParams {
-    pub sender: Address,
+    /// Address of the sender or operator that initiated the transfer
+    pub sender: ActorID,
```

Comment:
Discussion: in this context the term "sender" is easily confusable with "from". This makes me think we should adopt the term "operator" throughout (with a slightly different meaning that ERC-777).